src/server/graphql/location/location-resolvers.ts

```diff
@@ -0,0 +1,13 @@
+import { Context } from '../../types/context';
+import { Resolvers } from '../types';
+
+export const locationResolvers: Resolvers<Context> = {
+	Query: {
+		location: async (_parent, args, context) => {
+			const locationOutput = await context.dataSources.locationDataSource.fetchLocationInfo(args.input);
+			return context.locationModelConverter.fromLocationQueryOutputToGqlLocations(locationOutput);
```

Comment:
What are your thoughts on using context variables over using direct references?

Reply:
Actually for the same reason as before, basically to get the benefits of dependency injection

Reply:
Oops, I think this one was a duplicate 😅
Thanks for answering both 😄

Reply:
No problem 😄